docs/deploying.rst

```diff
@@ -51,6 +51,8 @@ as a setuptools distribution.
 Hot-deploying Toil
 ------------------
 
+.. todo:: "Hot-deploying" is a confusing name. What is better?
+
 Toil can be easily deployed to a remote host, given that both Python and Toil
 are present. The first order of business after copying your workflow to each
```

Comment:
I find this sentence confusing.  Where are Python and Toil present?  This seems to imply that they are on the remote host.  If they are already on the remote host, why do I need to deploy them to the remote host?

I am thinking deploy == remote install

Then it jumps into copying my workflow.   What happened to deploy Toil.  My intuition is that one would either deploy toil then deploy my workflow or deploy both toil and the workflow together.